### PR TITLE
coinbase layerswap deposit

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

**Comment by jonator** [comment]
Oooh this is cool

**Comment by jonator** [comment]
We need to make our onramps more visible. I'm thinking when the user clicks "Buy Tokens" in corner, it doesn't just show Transak (USD>OSMO), but they can select a list of tokens to buy (currently OSMO,USDC), then the list of methods (Transak for OSMO, Kado, Coinbase (via Layerswap) for USDC..

This is in addition to the deposit workflows we are maintaining

File: packages/web/config/ibc-assets.ts [hunk 1]
```diff
@@ -44,7 +44,9 @@ export const IBCAssetInfos: (IBCAsset & {
             AxelarSourceChainConfigs.usdc.fantom,
           ],
         },
-        fiatRamps: [{ rampKey: "kado" as const, assetKey: "USDC" }],
+        fiatRamps: [
+          { rampKey: "layerswapcoinbase" as const, assetKey: "USDC" },
```
Comment (jonator) [inline packages/web/config/ibc-assets.ts:48]: Removing kado?

Comment (sunnya97) [reply packages/web/config/ibc-assets.ts:48]: Yeah.  It's not really a way of depositing USDC.  It's an onramp for USDC, and should instead be moved to a fiat onramp selector along with transak

Comment (jonator) [reply packages/web/config/ibc-assets.ts:48]: Oh ok. We we're considering a move from bank account > osmosis as depositing, along with bridges/sends from CEXs

**Comment by sunnya97** [comment]
@jonator Yep!  Started working on that in #1358

File: packages/web/integrations/layerswap/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+> = ({}) => {
+  const { chainStore, accountStore } = useStore();
+
+  const account = accountStore.getAccount(chainStore.osmosis.chainId);
+
+  if (!(account.walletStatus === WalletStatus.Loaded)) return null;
+
+  return (
+    <iframe
+      src={`https://www.layerswap.io/?destNetwork=osmosis_mainnet&sourceExchangeName=Coinbase&destAddress=${account.bech32Address}&addressSource=osmosiszone&lockNetwork=true`}
```
Comment (sunnya97) [inline packages/web/integrations/layerswap/index.tsx:18]:
```suggestion
      src={`https://www.layerswap.io/?destNetwork=osmosis_mainnet&sourceExchangeName=Coinbase&destAddress=${account.bech32Address}&addressSource=osmosiszone&lockNetwork=true&products=CEX_TO_NETWORK`}
```

File: packages/web/public/logos/coinbase.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+<svg width="800px" height="800px" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg">
+   <circle cx="512" cy="512" r="512" style="fill:#0052ff"/>
```
Comment (jonator) [inline packages/web/public/logos/coinbase.svg:3]: This logo looks off.. where did you get it? Is there a way we can get an official logo?

Comment (sunnya97) [reply packages/web/public/logos/coinbase.svg:3]: From here:  https://www.svgrepo.com/svg/331345/coinbase-v2

Comment (sunnya97) [reply packages/web/public/logos/coinbase.svg:3]: I can switch it for this one:  https://logosarchive.com/wp-content/uploads/2021/12/Coinbase-icon-symbol-1.svg

Comment (sunnya97) [reply packages/web/public/logos/coinbase.svg]: This one has the proper logo

**Review by jonator — APPROVED** [review]

**Pull request merged**